### DISCUSSION TITLE
How can I load mtl files for materials of an obj file?

### DISCUSSION THREAD

**Discussion by F3RN4NDOJF3RN4ND3Z** [StereoKit/StereoKit#234] [Q&A] [answered]
I am adding an obj model and it is loading fine but with no materials, it is just rendering as white how can I add the materials?

**Comment by maluoi** [comment, marked as answer]
The obj format is a really old, poorly specified format, and StereoKit's support for it is somewhat basic. It does not include support for material files at the moment.

If you have the ability to, I'd recommend using .gltf or .glb files. Are you creating these models yourself, or are you acquiring them as .obj from somewhere else?

**Reply by F3RN4NDOJF3RN4ND3Z** [reply]
I get it from cgtrader ... So I decided to export them as gltf
